packages/cli/src/lib/monorepo/PackageGraph.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Package } from '@manypkg/get-packages';
+
+type PackageJSON = Package['packageJson'];
+
+export interface ExtendedPackageJSON extends PackageJSON {
+  scripts?: {
+    [key: string]: string;
+  };
+  // The `bundled` field is a field known within Backstage, it means
```

Comment:
Sounds like a candidate for lifting into what comes out of #8729

Reply:
At which point a `BackstagePackageJsonExtensions` type in `@backstage/types` might be interesting

Reply:
Yep idd. Perhaps not `@backstage/types` though as it's really only interesting for tooling, probably `@backstage/cli-common`

Reply:
Ah yep, makes sense